Controllers/ExportController.cs

```diff
@@ -30,12 +30,16 @@ public class ExportController : Controller
         #region Dependencies
 
         private readonly IAuthorizationService _authorizationService;
+        private readonly IExportService _exportService;
+        private readonly ILogger _logger;
+        private readonly INotifier _notifier;
         private readonly ISession _session;
         private readonly ISiteService _siteService;
 
         #region Properties
 
         private dynamic New { get; }
+        public IHtmlLocalizer TH { get; set; }
```

Comment:
`TH` instead of `T`?

Reply:
Something they do in Orchard Core, it's an `IHtmlLocalizer` not an `IStringLocalizer`.